scripts/sct_image.py

```diff
@@ -276,12 +276,12 @@ def main(args=None):
                 fname_out.append(add_suffix(fname_in[0], '_' + dim_list[dim].upper() + str(i).zfill(4)))
                 im.setFileName(fname_out[i])
                 im.save()
+        # display viewer syntax
+        sct.display_viewer([str(fname_out)])
```

Comment:
Something smells with `fname_out` here.

Reply:
yes i had the same feeling-- didn't have time to investigate further...

Reply:
fixed in 7624845cf4fbe725361a48a6fcf22835e429f617